.travis.yml

```diff
@@ -15,6 +15,7 @@ before_install:
   # setup https authentication credentials, used by ./mvnw release:prepare
   - git config credential.helper "store --file=.git/credentials"
   - echo "https://$GH_TOKEN:@github.com" > .git/credentials
+  - openssl aes-256-cbc -K $encrypted_0d0e64b78c84_key -iv $encrypted_0d0e64b78c84_iv -in travis/zipkin-gcp-ci-0d7917f58da7.json.enc -out travis/zipkin-gcp-ci-0d7917f58da7.json -d
```

Comment:
need an addition to the RELEASE.md about this and a summary comment about what's going on and any values sourced from implicit environment. ex a link to travis where the value is set in case the repo moves, travis has a problem, etc.

Reply:
added more instructions and info, ptal